README.md

```diff
@@ -1,57 +1,99 @@
 
-### Prerequisites
+These sample apps back the [Tutorial: enable authentication in App Service and access storage and Microsoft Graph](https://docs.microsoft.com/azure/app-service/scenario-secure-app-overview) articles on docs.microsoft.com.
 
-(ideally very short, if any)
+## Configure authentication for a web app and limit access to users in your organization
 
-- OS
-- Library version
-- ...
+If you are a Line of Business (LOB) developer, you'd probably want to only sign-in users in your organization with their work or school accounts. In this step, you learn how to enable the Azure App Service authentication/authorization module in Azure App Service, so you can sign in users and access data by writing minimal or no code in your web app. This step is required and is done inline with the other steps. Read [Add authentication to your web app running on Azure App Service](https://docs.microsoft.com/azure/app-service/scenario-secure-app-authentication-app-service) for more information.
 
-### Installation
+## Securely access Azure storage on behalf of the web application using managed identities​
+
+In this step, you configure authentication for a web app and [access Azure blob storage as the web app (using managed identities)](https://github.com/Azure-Samples/ms-identity-easyauth-nodejs-storage-graphapi/tree/main/1-WebApp-storage-managed-identity).  
+
+You learn how to:
+
+- Setup authentication for your web app.
+- Create a system-assigned managed identity on a web app.
+- Access storage from a web app (on behalf of the app) by using managed identities
+
+## Access data in Microsoft Graph on behalf of the signed-in user​
+
+In this step, you configure authentication for a web app and [call Microsoft Graph from a secured app on behalf of the signed-in user](https://github.com/Azure-Samples/ms-identity-easyauth-nodejs-storage-graphapi/tree/main/2-WebApp-graphapi-on-behalf).
+
+You learn how to:
+
+- Setup authentication for your web app.
+- Grant delegated permissions to a web app.
+- Call Microsoft Graph from a web app on behalf of the signed-in user.
 
-(ideally very short)
+## Access data in Microsoft Graph on behalf of the web application using managed identities​
 
-- npm install [package name]
-- mvn install
-- ...
+In this step, you configure authentication for a web app and [call Microsoft Graph as the web app (using managed identities)](https://github.com/Azure-Samples/ms-identity-easyauth-nodejs-storage-graphapi/tree/main/3-WebApp-graphapi-managed-identity)
 
-### Quickstart
-(Add steps to get up and running quickly)
+You learn how to:
+
+- Setup authentication for your web app.
+- Create a system-assigned managed identity on a web app.
+- Add Microsoft Graph API permissions to a managed identity.
+- Call Microsoft Graph from a web app (on behalf of the app) by using managed identities.
```

Comment:
is this section redundant?

Reply:
No this list matches the [backing tutorial's sections](https://docs.microsoft.com/en-us/azure/app-service/scenario-secure-app-overview)